atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraGetCandidateCellsForSweepingTest.java

```diff
@@ -38,4 +44,48 @@ protected KeyValueService createKeyValueService() {
                 CassandraContainer.LEADER_CONFIG,
                 Mockito.mock(Logger.class));
     }
+
+    @Test
+    public void returnCandidateIfPossiblyUncommittedTimestamp() {
+        new TestDataBuilder().put(1, 1, 10L).store();
+        assertThat(getAllCandidates(conservativeRequest(PtBytes.EMPTY_BYTE_ARRAY, 40L, 1)))
+                .containsExactly(ImmutableCandidateCellForSweeping.builder()
+                        .cell(cell(1, 1))
+                        .sortedTimestamps(new long[] { 10L })
+                        .isLatestValueEmpty(false)
+                        .numCellsTsPairsExamined(1)
+                        .build());
+    }
+
+    @Test
+    public void returnCandidateIfTwoCommittedTimestamps() {
+        new TestDataBuilder().put(1, 1, 10L).put(1, 1, 20L).store();
+        assertThat(getAllCandidates(conservativeRequest(PtBytes.EMPTY_BYTE_ARRAY, 40L, 1)))
+                .containsExactly(ImmutableCandidateCellForSweeping.builder()
+                        .cell(cell(1, 1))
+                        .sortedTimestamps(new long[] { 10L, 20L })
+                        .isLatestValueEmpty(false)
+                        .numCellsTsPairsExamined(2)
+                        .build());
+    }
+
+    // TODO(nziebart): check that this is ok to fail, and delete
```

Comment:
Looks like its OK to fail now that `minUncommitedTimestamp` is no longer a thing when getting the candidate cells. https://github.com/palantir/atlasdb/blob/develop/atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepableCellFilter.java#L95-L101 will ensure that the latest commited value is not swept.

Reply:
yep, @fsamuel-bs looked at this with the same conclusion. filtering it in the shim is just an optimization